src/main/java/hellos/Hellos.java

```diff
@@ -19,6 +20,9 @@ private String nicSaysHello() {
     return "Nic says 'Howdy!'\n";
   }
 
+  private String nateSaysHello() {
+    return "Nate says 'Hola!'\n";
+
   private String kkSaysHello() {
     return "KK says 'Hello!'\n";
   }
```

Comment:
You're missing the closing brace after your greeting, which is why this won't compile and run.

```suggestion
  private String nateSaysHello() {
    return "Nate says 'Hola!'\n";
  }
  
  private String kkSaysHello() {
    return "KK says 'Hello!'\n";
  }
```